sct-commons/src/main/java/org/lfenergy/compas/sct/commons/LnService.java

```diff
@@ -60,33 +59,37 @@ public Optional<TLN> findLn(TLDevice tlDevice, Predicate<TLN> lnPredicate) {
      * @return the Lnode Status
      */
     public ActiveStatus getLnStatus(TAnyLN tAnyLN, LN0 ln0) {
-        Optional<ActiveStatus> ln0Status = getDaiModStval(ln0);
-        return getDaiModStval(tAnyLN).filter(ActiveStatus.OFF::equals).orElseGet(() -> ln0Status.orElse(ActiveStatus.OFF));
+        Optional<ActiveStatus> ln0Status = getDaiModStValValue(ln0);
+        return getDaiModStValValue(tAnyLN).filter(ActiveStatus.OFF::equals).orElseGet(() -> ln0Status.orElse(ActiveStatus.OFF));
     }
 
-    public Optional<ActiveStatus> getDaiModStval(TAnyLN tAnyLN) {
+    public Optional<ActiveStatus> getDaiModStValValue(TAnyLN tAnyLN) {
```

Comment:
Methods like `getDaiModStValValue`  and `getDaiModStValValue` can be defined in the interface that LnService implement.
```java
Optional<TDAI> getDaiModStVal(TAnyLN tAnyLN)
Optional<TDAI> getDaiModStValValue(TAnyLN tAnyLN)
```

Reply:
OK